telegram/ext/picklepersistence.py

```diff
@@ -102,7 +104,7 @@ class PicklePersistence(BasePersistence[UD, CD, BD]):
     @overload
     def __init__(
         self: 'PicklePersistence[Dict, Dict, Dict]',
-        filename: str,
+        filename: Union[Path, str],
```

Comment:
okay, if you only want to change the internal attribute and leave the parameter, then you replaced the wrong name in the doc string.

But, why?

Reply:
replaced all to ```filepath```